ocaml/xapi-consts/api_errors.ml

```diff
@@ -1395,3 +1395,7 @@ let telemetry_next_collection_too_late =
 let illegal_in_fips_mode = add_error "ILLEGAL_IN_FIPS_MODE"
 
 let too_many_groups = "TOO_MANY_GROUPS"
+
+let invalid_cache_size = add_error "INVALID_CACHE_SIZE"
```

Comment:
If you introduce new errors here, then you need to also define them in datamodel_errors.ml for the SDK to pick them up.

An alternative is to use the existing `invalid_value` error (rather than define new errors), which takes two parameters:
```
  error Api_errors.invalid_value ["field"; "value"]
    ~doc:"The value given is invalid" () ;
```